src/js/actions/tools.js

```diff
@@ -486,8 +488,7 @@ define(function (require, exports) {
             "toolRectangle.select",
             "toolSampler.select", "toolSampler.deselect",
             "toolType.select", "toolType.deselect"
-        ],
-        modal: true
```

Comment:
Also, this used to be marked modal-safe, but also started by calling `endModalToolState`. But that's already the default behavior, so I removed both.